### PR TITLE
Add periodic job: periodic-prow-image-autoowners

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

**Comment by petr-muller** [comment]
I don't think autoowners is ready for the show-time yet: 

1. https://github.com/openshift/ci-tools/blob/master/cmd/autoowners/README.md says: _If the upstream did not have an OWNERS file, the utility removes the associated OWNERS_. This is IMHO not desirable: in openshift/release we enforce OWNERS for good reason, we should not sync this bad state from the upstream
2. Like mentioned in the JIRA, some teams intentionally have owners here different from the upstream owners (ci owners != component owners 1:1) and we should be able to support this scenario by providing an opt-out mechanism from the sync.

**Comment by petr-muller** [comment]
/hold
...for reasons stated above.

**Comment by hongkailiu** [comment]
@petr-muller 

`whitelist` is supported already while it is not in the readme yet.

https://github.com/openshift/ci-tools/pull/135/files

**Comment by hongkailiu** [comment]
https://github.com/openshift/ci-tools/pull/139 addresses item 1 in @petr-muller comment.

**Comment by hongkailiu** [comment]
https://github.com/openshift/ci-tools/pull/139 is in ...
doing rehearsal now.

**Comment by hongkailiu** [comment]
prowjob for rehearsal:
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/periodic-prow-image-autoowners/1

PR created from the above job:
https://github.com/openshift/release/pull/5054

**Comment by hongkailiu** [comment]
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/periodic-prow-image-autoowners/8
looks like right.

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -308,6 +308,71 @@ periodics:
...
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: "30 10 * * 1"  # Run at 10:30 UTC each Monday
```
Comment (stevekuznetsov) [inline ci-operator/jobs/infra-periodics.yaml]: Wow, we're _really_ automating @petr-muller

Comment (petr-muller) [reply ci-operator/jobs/infra-periodics.yaml]: :stuck_out_tongue:

**Pull request merged**